net/message/blockHdr.go

```diff
@@ -229,6 +229,10 @@ func GetHeadersFromHash(startHash common.Uint256, stopHash common.Uint256) ([]le
 				return nil, 0, err
```

Comment:
1. when underflow occurs, the error message "leveldb not found" is misleading, 
     shifting our focus to the chainstore module. 
2. performance, loading and deserializing data from db is expensive. for example
    curHeight=20, stopHeight=30, headerHeight = 40, when enter the for loop, it 
    tries to load headers 31~2030, error occurs at header 41 after loading header 31~40.